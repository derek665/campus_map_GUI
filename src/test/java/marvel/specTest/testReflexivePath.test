# test same node on findPath

LoadGraph testReflexive soccer.tsv

FindPath testReflexive Arsenal Arsenal