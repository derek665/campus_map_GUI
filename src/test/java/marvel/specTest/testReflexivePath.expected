# test same node on findPath

loaded graph testReflexive

path from Arsenal to Arsenal: